examples-standalone/grid-charts-integration/src/app/components/stock-list/stock-list.component.html

```diff
@@ -0,0 +1,106 @@
+<div class="k-custom-notification">
```

Comment:
That approach seems better, than using the Notification service. Still, using the service to open the notification programmatically is possible.